Editor/UI/Windows/CourseWindow.cs

```diff
@@ -231,15 +231,18 @@ private void DrawChapterWorkflow(Rect scrollRect)
 
         private void OnSceneOpened(Scene scene, OpenSceneMode mode)
         {
-            if (RuntimeConfigurator.Exists == false)
+            if (BuildPipeline.isBuildingPlayer == false && RuntimeConfigurator.Exists == false)
```

Comment:
Shouldn't this be an "or" case?

Reply:
No, the idea is to close the window only when a new empty scene is opened and Unity is not building.

For some reason, Unity opens a new scene when running unit test and when building, after it finishes those proceses it goes back to the original scene 🤷‍♂️🤦‍♂️